### PR TITLE
Fix crash when opening 4xx pages with no description.

### PR DESCRIPTION
**Pull request by bimusiek**
Hey, at https://freeyourmusic.com we have a weird case, where user is redirected to a page with 404 error that crashes on iOS. After further investigation, we found that this is because of `description=NULL` as seen on screenshot:
CleanShot 2024-04-25 at 15 24 05@2x

This PR fixes this issue and possibly https://github.com/react-native-webview/react-native-webview/issues/3412

### CODE REVIEW

File: apple/RNCWebView.mm [hunk 1]
```diff
@@ -109,7 +109,7 @@ - (instancetype)initWithFrame:(CGRect)frame
                     .lockIdentifier = [[dictionary valueForKey:@"lockIdentifier"] doubleValue],
                     .title = std::string([[dictionary valueForKey:@"title"] UTF8String]),
                     .code = [[dictionary valueForKey:@"code"] intValue],
-                    .description = std::string([[dictionary valueForKey:@"description"] UTF8String]),
+                    .description = std::string([[dictionary valueForKey:@"description"] UTF8String] ?: ""),
```
Comment (DorianMazur) [inline apple/RNCWebView.mm:112]: You need to do the same on line 233.

**Comment by DorianMazur** [comment]
@Titozzz @jamonholmgren Can you check this? This is important PR and I think should be merged after applying change suggested in my comment.

File: apple/RNCWebView.mm (hunk 1 above)
Comment (bimusiek) [reply apple/RNCWebView.mm:112]: Done.

**Comment by bimusiek** [comment]
@chiaramooney @Titozzz I would love to have this merged. We still use simple patch now, but this is quite common crash for any non 200 page that does not return description. Also, as reported by someone else, this is also crashing for Cloudflare protected pages in case of 403 there.

Seemed like quite nice bug fix, but it is more than a month now without any response or comment? :)

**Review by TheAlmightyBob — APPROVED** [review]

**Comment by TheAlmightyBob** [comment]
@bimusiek It's saying "This branch is out-of-date with the base branch" but not giving me any option to rebase it (don't think I've seen GitHub do that before...?). Are you able to update this branch?

**Pull request merged**